### DISCUSSION TITLE
A tutorial about the highlights of the browser and focus modes of the Navigation Block

### DISCUSSION THREAD

**Discussion by tinjure20** [WordPress/developer-blog-content#137] [Topic Ideas]
Since my first contribution just got [published](https://developer.wordpress.org/news/2023/06/the-style-book-a-one-stop-shop-for-styling-block-themes/), I have begun playing around with a few new features that are anticipated to be included in the WordPress 6.3 release. 

There has been [significant progress](https://github.com/WordPress/gutenberg/issues/50396#issuecomment-1594836796) on the Navigation Block recently. I have been looking at a few features, such as browser mode ([#50396](https://github.com/WordPress/gutenberg/issues/50396)), focus mode ([#39286](https://github.com/WordPress/gutenberg/pull/39286)), and [related](https://github.com/WordPress/gutenberg/pull/51492) features of the Nav block. Even though I have not yet thoroughly researched all the related GitHub tickets, I wanted to put the idea out there early for discussion to see if it would be a good topic for a tutorial. 

The workflow will become more detailed as the Nav Block enters 6.3 Beta and its dev notes draft is available. If the topic is approved during the upcoming meeting and assigned to me, I hope to complete the first draft for review around the 6.3 release date.

**Resources**
- [Navigation on Browse Mode #50396 ](https://github.com/WordPress/gutenberg/issues/50396) 
- [Add focus mode for Navigation Menus #39286](https://github.com/WordPress/gutenberg/pull/39286)
- [Navigation on Browse Mode #50396](https://github.com/WordPress/gutenberg/issues/50396) 
- [Site Editor: Add Library for Template Parts & Patterns Management #51078 ](https://github.com/WordPress/gutenberg/pull/51078)
- [Navigation 6.3 Priorities](https://github.com/WordPress/gutenberg/issues/50165) 
-  [Dev Notes tracking issue for WordPress 6.3](https://github.com/WordPress/gutenberg/issues/51484)

**Comment by bph** [comment]
The topic has been approved. The discussion is now locked,
Please continue the conversation on the issue
- #149

**Comment by bph** [comment]
Closing this for now. We can always reopen if there is a need for a developer relevant post about the Navigation block. Or start a new discussion.
